just_audio_web/lib/just_audio_web.dart

```diff
@@ -961,10 +961,15 @@ class _PlayPauseQueue {
 
   Future<void> _run() async {
     await for (var request in _queue.stream) {
-      if (request.playing) {
-        await audioElement.play();
-      } else {
-        audioElement.pause();
+      try {
+        if (request.playing) {
+          await audioElement.play();
+        } else {
+          audioElement.pause();
+        }
+      } catch (err) {
+        request.completer.completeError(err);
+        continue;
```

Comment:
Rather than use `continue`, this can be handled by the try/catch control ostructure by including the normal `complete()` from below within the `try` block. I can fix this while merging.